src/app/application.cpp

```diff
@@ -403,15 +403,63 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    bool skipTorrentDialog = false;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user. Use of
+        // any of these should prevent the "Add new torrent" dialog from
+        // opening:
+
+        if (param.startsWith("@savePath=")) {
+            torrentParams.savePath = param.mid(10);
+            skipTorrentDialog = true;
```

Comment:
Sorry, I don't understand your point. I thought that it would be sufficient just add the option and in case of `--skip-dialog=false` and any of the new options set just abort and show that this combination of options is not yet implemented.